.github/workflows/release.yml

```diff
@@ -2,9 +2,8 @@ name: Release
 
 on:
   push:
-    branches:
-      - v[0-9]+.x
-
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]*'
```

Comment:
why the last `*` and not `+`? 
is `v2.1.` a valid tag to release?

Reply:
yes, that was the proper regex but I changed it in order to test it with `v2.0.0-pre` and don't edit the package.json unnecessarily. I'm changing it back